bugsnag-android-core/src/main/java/com/bugsnag/android/SessionTracker.java

```diff
@@ -320,11 +322,11 @@ DeliveryStatus deliverSessionPayload(Session payload) {
     }
 
     void onActivityStarted(String activityName) {
-        updateForegroundTracker(activityName, true, System.currentTimeMillis());
+        updateForegroundTracker(activityName, true, SystemClock.elapsedRealtime());
```

Comment:
This will also need a fix in `updateForegroundTracker` which tracks the session start as a `Date` using this value:
```java
startNewSession(new Date(nowMs), client.getUser(), true);
```

Reply:
Updated to use `new Date()` instead.